01-architecture.md

```diff
@@ -8,27 +8,27 @@ This documents provides an overview of the architecture of the FVM, including a
 
 The FVM aims to (a) support a multitude of programming models for actors and (b) facilitate the onboarding of smart contracts and programs written for other environments, so they can leverage the storage capabilities of the Filecoin network.
 
-Its architecture is inspired by the [hypervisor](https://en.wikipedia.org/wiki/Hypervisor) concept from the world of Virtual Machines. It is not however _a_ hypervisor architecture, as user-space actors run directly on top of the native layer.
+Its architecture is inspired by the [hypervisor](https://en.wikipedia.org/wiki/Hypervisor) concept from the world of Virtual Machines. The abstraction is not translated directly, though.
```

Comment:
This may not be the right place to ask, but I'm in here anyway. Has there been thought put into cross-chain interfaces for the FVM and hypervisor?

Reply:
The declaration of intentions here is that we want to make chain interoperability a native feature insofar possible. However, we haven't prioritised this yet, and would love to get your thoughts on it ;-)